website_sale_osc/README.rst

```diff
@@ -0,0 +1,46 @@
+One Step Checkout
+=================
+
+Provide an All-In-One Checkout for Your Odoo Customer
+
+.. image:: https://raw.githubusercontent.com/blooparksystems/website_sale_osc/master/static/description/osc.png
+
+The One Step Checkout brings different options:
+
+.. image:: https://raw.githubusercontent.com/blooparksystems/website_sale_osc/master/static/description/settings.png
+
+One Step Checkout combines all Odoo Checkout steps into one and removes all unnecessary fields and 
+questions. Never before has check- out been easier and faster!
+
+Improving the checkout process results in more customers completing their sales, and this has an immediate impact on your bottom line. It is the single most effective technical change you can make to reduce shopping cart abandonment.
+
+Regards Bloopark
+
+
+Known issues / Roadmap
+======================
+
+ * For now, this add-on will add a One Step Checkout to the website_sale add-on.
```

Comment:
In this section you have to put possible things to do. I can figure out that this module is not going to be compatible with _website_event_register_free_with_sale_. Can you check it or at least mention it?